File: app/system/modules/ByodConfigModule.java
package system.modules;

import com.google.inject.AbstractModule;
import util.config.ByodConfigHandler;
import util.config.ByodConfigHandlerImpl;

public class ByodConfigModule extends AbstractModule {

    @Override
    protected void configure() {
        bind(ByodConfigHandler.class).to(ByodConfigHandlerImpl.class);
    }
}

File: app/system/modules/ByodConfigModule.scala
package system.modules

import com.google.inject.AbstractModule
import util.config.ByodConfigHandler
import util.config.ByodConfigHandlerImpl

class ByodConfigModule extends AbstractModule:
  override def configure(): Unit = bind(classOf[ByodConfigHandler]).to(classOf[ByodConfigHandlerImpl])
